src/database/models/travel_request.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (sequelize, DataTypes) => {
+  const travel_request = sequelize.define('travel_requests', {
+    user_id: DataTypes.INTEGER,
+    travel_type: {
+      type: DataTypes.ENUM,
+      values: ['one-way', 'return']
+    },
+    origin: DataTypes.TEXT,
+    destination: DataTypes.TEXT,
+    departure_date: DataTypes.DATE,
+    return_date: DataTypes.DATE,
+    travel_purpose: DataTypes.TEXT,
+    accommodation_id: DataTypes.INTEGER,
+    approval_status: {
+        type: DataTypes.ENUM,
+        values: ['accepted', 'pending', 'rejected'],
+        defaultValue: 'pending'
+    },
+    multi_city: DataTypes.BOOLEAN
+  }, {});
+  travel_request.associate = (models) => {
+    travel_request.belongsTo(models.users, {
+      foreignKey: 'user_id',
+      onDelete: 'CASCADE'
+    });
+
+    travel_request.belongsTo(models.accommodations, {
```

Comment:
I don't think this association is necessary. Deleting the entries in the accommodation table shouldn't automatically delete any travel request.

Reply:
@chuksjoe .. You are right about that.. but the association references a particular accommodation, and for the same reason you stated. if the accommodation detail is modified or deleted. **postgres will scream**. I think the application needs that